executor/common_linux.h

```diff
@@ -206,6 +206,37 @@ static int netlink_send(struct nlmsg* nlmsg, int sock)
 	return netlink_send_ext(nlmsg, sock, 0, NULL);
 }
 
+// TODO: replace other similar methods by netlink_query_family_id
+static int netlink_query_family_id(struct nlmsg* nlmsg, int sock, const char* family_name)
+{
+	struct genlmsghdr genlhdr;
+	memset(&genlhdr, 0, sizeof(genlhdr));
+	genlhdr.cmd = CTRL_CMD_GETFAMILY;
+	netlink_init(nlmsg, GENL_ID_CTRL, 0, &genlhdr, sizeof(genlhdr));
+	netlink_attr(nlmsg, CTRL_ATTR_FAMILY_NAME, family_name, strnlen(family_name, GENL_NAMSIZ - 1) + 1);
+	int n = 0;
```

Comment:
If you do the refactor to switch to this generic one, I'd really make the generic one as identical to the ones we already have as possible, that includes leaving the variable declarations as-is (unless this 'n' needs to be initialized to 0) and at the start of the function. Essentially we should just be able to put them side-by-side and see no diff except the family_name.

Reply:
The new generic function is idential to the functions it replaces.